src/components/balance/Address.vue

```diff
@@ -121,8 +69,8 @@
         class="
           tw-border-l tw-border-gray-100
           dark:tw-border-darkGray-600
-          tw-flex tw-items-center tw-px-1
-          md:tw-px-2
+          tw-flex tw-items-center tw-pl-1
+          md:tw-pl-2
         "
       >
         <a :href="subScan" target="_blank" rel="noopener noreferrer">
```

Comment:
H160 accounts cannot be viewed on Subscan. We may have a EVM explorer in the near future by a different provider, but for now, we cannot view it.

Reply:
Oh I didn't know that! It must have been my account then. Thanks!